lib/broadside/target.rb

```diff
@@ -93,5 +93,16 @@ def to_h
         Cluster: @cluster
       }
     end
+
+    def update_safe_service_config
```

Comment:
i mean, this is where cute ruby gets _close_ to english, but isn't english. the intent here is:

service config
vs.
update-safe service config

not "update service config, which is safe"